.github/workflows/scripts/scanner-get-released-tags.sh

```diff
@@ -10,6 +10,8 @@ tags=$(mktemp)
 git tag > "$tags"
 
 # Function to resolve the tag associated with a vulnerability updater version.
+# This logic is duplicated from scanner-output-release-versions.sh. An optimization is needed to eliminate this duplication.
```

Comment:
Currently `scanner-output-release-versions.sh` is dependent on `release_version` file in scanner but it should not be the dependency for the new workflow.